schemas/com.snowplowanalytics.snowplow/media_player_ad/jsonschema/1-0-0

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity with information about the currently played ad",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_ad",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "Friendly name of the ad.",
+      "type": [
+        "null",
+        "string"
+      ],
+      "maxLength": 4096
+    },
+    "adId": {
+      "type": "string",
+      "format": "uuid",
+      "description": "Unique identifier for the ad."
+    },
+    "creativeId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 4096,
+      "description": "Unique identifier for the ad."
+    },
+    "podPosition": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "The position of the ad within the ad break, starting with 1.",
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "duration": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Length of the video ad in seconds.",
+      "minimum": 0,
+      "maximum": 9007199254740991
```

Comment:
Could we be more explicit and make these integers?

Reply:
I wanted to make this have the same format as the `duration` property in the `media_player` schema with info about the main media content. That one allows for double precision values, so I thought we should do the same here.

Reply:
Sorry I meant the `podPosition` instead?